sdk/servicebus/Azure.Messaging.ServiceBus/tests/Message/MessageLiveTests.cs

```diff
@@ -178,5 +179,116 @@ private class TestBody
             public int B { get; set; }
             public bool C { get; set; }
         }
+
+        [Test]
```

Comment:
Can we add a test where the user sends a message through normal ctors, but checks received message using AMQP extensions?

Reply:
That's a great idea. Thanks! I'll add it.